### ISSUE TITLE
Stop when RRULE[COUNT] is reached

### ISSUE THREAD

**Issue by jeromecombes** [johngrogg/ics-parser#69]
If we have a weekly recurrence with 2 days in weeks and the count parameter set to 4, the function will count 4 weeks and will return 8 events, while it should return 4 events.

Example: 
```
BEGIN:VEVENT
DTSTART;TZID=Europe/Paris:20160705T090000
DTEND;TZID=Europe/Paris:20160705T100000
RRULE:FREQ=WEEKLY;COUNT=4;BYDAY=TU,SU
DTSTAMP:20160514T170815Z
UID:tl6n0s54hq71d0hg3fm4zzzzt0@google.com
CREATED:20150624T141227Z
DESCRIPTION:
LAST-MODIFIED:20160707T075813Z
LOCATION:
SEQUENCE:1
STATUS:CONFIRMED
SUMMARY:TEST WKST MO
TRANSP:OPAQUE
END:VEVENT
```

**Comment by u01jmg3** [comment]
See https://github.com/johngrogg/ics-parser/pull/66/commits/88650ef8721ff1eedfbbfd9ed9bf7fb83b13a0d3

**Comment by u01jmg3** [comment]
Fixed by PR #70
